app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
```

Comment:
for this - the curly braces must be located in one of two options: open and start with the text or have a line break between the text.